torch/fx/experimental/GraphManipulation.py

```diff
@@ -0,0 +1,7 @@
+def replace_all_uses_with(fx_module, old_op, old_target, new_op, new_target):
```

Comment:
Style nit, we're trying to be conscientious about adding Python type annotations in the FX codebase where we can :). Can you annotate these arguments?